docs/user/dashboard/url-drilldown.asciidoc

```diff
@@ -99,7 +99,7 @@ https://github.com/elastic/kibana/issues?q={{event.value}}
 
 A Handlebars expression is a `{{`, some contents, followed by a `}}`. When the drilldown is executed, these expressions are replaced by values from the dashboard and interaction context.
 
-Refer to Handlebars https://handlebarsjs.com/guide/expressions.html#expressions[documentation] to learn about advanced use cases.
+Refer to Handlebars https://ela.st/handlebars-docs#expressions[documentation] to learn about advanced use cases.
 
 [[helpers]]
 In addition to https://handlebarsjs.com/guide/builtin-helpers.html[built-in] Handlebars helpers, you can use the following custom helpers:
```

Comment:
Shouldn't we do the same to this link? It seems like it's still working, but we could easily run into the same issue.

Reply:
You are def right, changing this!